Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM nvidia/cuda:10.1-devel-ubuntu18.04
```

Comment:
You might want to use this image only for building (and not as a runtime image).
```suggestion
FROM nvidia/cuda:10.1-devel-ubuntu18.04 AS build
```

Then to run the app you could use a lighter base image as follows:
```docker
# Right after building ethminer

FROM nvidia/cuda:10.1-base-ubuntu18.04

COPY --from=build /usr/local/bin/ethminer /usr/local/bin/
```